devel/build-to-localhost

```diff
@@ -46,8 +47,8 @@ docker buildx build \
     --cache-from $BASE_BUILDER_IMAGE \
     --cache-from $BASE_IMAGE \
     --cache-to type=inline \
-    --tag $BASE_BUILDER_IMAGE:$tag \
-    --load \
+    --tag localhost:5000/$BASE_BUILDER_IMAGE:$tag \
+    --push \
```

Comment:
`localhost:5000` or at least the port `5000` should be parameterized for customization across these scripts. That would make 3 parameters to this script, which is a bit messy if they're arbitrarily positional. Maybe it'd be good to consider `getopt` for named parameters.

Reply:
Used `getopts` in [force-push](https://github.com/nextstrain/docker-base/compare/a7a2ac6c0499e4f9adf6d89956329f0461b5dc67..7cec670719ad249f2088f34fc28d05ca3821e612).